src/workerd/util/sqlite.c++

```diff
@@ -236,6 +236,37 @@ static constexpr kj::StringPtr ALLOWED_SQLITE_FUNCTIONS[] = {
   "json_tree"_kj,
 };
 
+// https://www.sqlite.org/pragma.html
+static constexpr kj::StringPtr ALLOWED_READ_PRAGMAS[] = {
```

Comment:
I think it would be clearer to call this `ALLOWED_NO_ARG_PRAGMAS`, since that makes it clear exactly why we're separating these pragmas and exactly what we need to enforce about them.